api.md

```diff
@@ -0,0 +1,188 @@
+#### Example
+```javascript
+new Builder({
+  map: {
+    'a': 'b.js'
+  }
+});
+
+new Builder('scripts', 'config.js');
+```
+
+### builder.config(config[, saveForReset[, ignoreBaseURL]])
+`config`: An object conforming to the [config api] (https:/github.com/systemjs/systemjs/blob/master/docs/config-api.md)  
+`saveForReset`: Reload this config when builder.reset() is called  
+`ignoreBaseURL`: Don't use the baseURL property from this config  
+#### Example
+```javascript
+builder.config({
+  map: {
+    'a': 'b.js'
+  }
+});
+```
+
+### builder.loadConfig(configFile[, saveForReset[, ignoreBaseURL]])
+`configFile`: A systemjs config file conforming to the [systemjs config api] (https:/github.com/systemjs/systemjs/blob/master/docs/config-api.md)  
+`saveForReset`: Reload this config when builder.reset() is called  
+`ignoreBaseURL`: Don't use the baseURL property from this config  
+#### Example
+```javascript
+builder.loadConfig('config.js').then(() => {
+});
+```
+### builder.loadConfigSync(configFile[, saveForReset[, ignoreBaseURL]])
+Synchronous version of `builder.loadConfig()`  
+
+`configFile`: A systemjs config file conforming to the [systemjs config api] (https:/github.com/systemjs/systemjs/blob/master/docs/config-api.md)  
+`saveForReset`: Reload this config when builder.reset() is called  
+`ignoreBaseURL`: Don't use the baseURL property from this config  
+#### Example
+```javascript
+builder.loadConfigSync('config.js');
+```
+
+### builder.reset()
+Reset the builder config to its initial state, or the last saved config() state
+#### Example
+```javascript
+builder.reset();
+```
+
+### builder.bundle(tree[, outfile][, options])
+### builder.bundle(expression[, outfile][, options])
+Concatenate all modules in the tree or module tree expression and optionally write them out to a file  
+
+`tree`: A tree object as returned from builder.trace(), or one of the arithmetic functions  
+`expression`: A [module tree expression](#module-tree-expressions)  
+`outfile`: The file to write out the bundle to  
+`options`: Additional bundle options as outlined below  
+
+Returns a promise which resolves with the bundle content
+#### Bundle options
+`minify`: Minify source in bundle output _(Default:true)_  
+`mangle`: Allow the minifier to shorten non-public variable names _(Default:false)_  
+`sourceMaps`: Generate source maps for minified code _(Default:false)_  
+`lowResSourceMaps`:  When true, use line-number level source maps, when false, use character level source maps _(Default:false)_  
+`globalName`: When building a self-executing bundle, assign the bundle output to a global variable _(Default:null)_   
+`globalDeps`: When building a self-executing bundle, indicates external dependendencies available in the global context _(Default:{})_  
+`fetch`: Override the fetch function to retrieve module source manually _(Default:undefined)_
```

Comment:
There's a whole lot more than these :)

See https://github.com/systemjs/builder/blob/master/lib/builder.js#L336, https://github.com/systemjs/builder/blob/master/lib/builder.js#L469 and https://github.com/systemjs/builder/blob/master/lib/builder.js#L583.